### DISCUSSION TITLE
the page can't be displayed with katex formula

### DISCUSSION THREAD

**Discussion by zhuzhzh** [requarks/wiki#5982] [Error / Bug Report]
The issue is caused by the `{{`.  If I use `{ {` instead, the issue is gone.  Other markdown editors(vsc, ob...) don't have such issue.

To reproduce the issue, put the following code in the page.

```
$$
I_M={{25 \times Data} \over {32767 \times 1.675}} \approx 0.0004555 \times Data \ (A)
$$
```

**Comment by a2k-hanlon** [comment]
Duplicate of #5216. Bug still exists on latest version as of writing 2.5.300.
